packages/preview/illc-mol-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "illc-mol-thesis"
```

Comment:
Unless this package is also officially endorsed by the institute, I would like you to rename it to make its name more distinctive. If it is an official template, can you ask a member of the staff to send an email to `hello@typst.app` stating so, please?

Reply:
I am not sure the staff would be fine with making this official, as none of them are Typst users (that I know of), so I would still be the only package maintainer even after graduation.

Still, I will look into this and maybe have this endorsed via email. In case this fails, would `logical-master-thesis` be distinctive enough?

Reply:
Okay, we'll see how it goes.

And the potential new name would be fine in my opinion.

Reply:
It's up to you. If you prefer we can also merge this first version as soon as I get the email, and let you publish an update once you've ported the full template to Typst.

Reply:
Yeah, that would also work. It might take a while for the logo to be properly licensed and the email to be sent, but when this happens, I'd be very happy to add the missing logo licence and see this first version 0.1.0 released. Thank you!

Reply:
Converted the package to a template. I'd like a new review, if possible, since I might have messed something up while doing so.

You should be getting an email from an `@uva.nl` email address any day now.